gomplate_test.go

```diff
@@ -129,6 +130,38 @@ func TestHasTemplate(t *testing.T) {
 	assert.Equal(t, "bar", testTemplate(t, g, tmpl))
 }
 
+func TestMissingKey(t *testing.T) {
+	g := NewRenderer(Options{
+		MissingKey: "zero",
+	})
+	tmpl := `{{ .name }}`
+	assert.Equal(t, "<no value>", testTemplate(t, g, tmpl))
```

Comment:
When it's `zero` shouldn't it be a zero value? from the docs, `<no value>` is the result of setting the `missingkey=default`

Reply:
zero value for the interface{} is zero

`missionkey=default` prints `<no value>` always, but `missionkey=zero` prints `0` for the ints, for example

Reply:
Right - that's why I'm confused about this test - why is `<no value>` being printed when `missingkey=zero`?

Reply:
because zero value for nil is `nil` and it prints `<no value>`

Reply:
ahhh, ok, thanks for explaining @xxxcoltxxx 🙂